src/librustdoc/json/conversions.rs

```diff
@@ -677,6 +677,7 @@ impl FromWithTcx<ItemType> for ItemKind {
             TraitAlias => ItemKind::TraitAlias,
             ProcAttribute => ItemKind::ProcAttribute,
             ProcDerive => ItemKind::ProcDerive,
+            _ => unreachable!(),
```

Comment:
Please use the name instead of a wildcard.
```suggestion
            Generic => unreachable!(),
```

Reply:
:+1: